test/unit/local/steps/incomplete_fixer.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-env mocha */
+/* @flow */
+
+const _ = require('lodash')
+const path = require('path')
+
+const should = require('should')
+const Builders = require('../../../support/builders')
+const configHelpers = require('../../../support/helpers/config')
+const pouchHelpers = require('../../../support/helpers/pouch')
+const { ContextDir } = require('../../../support/helpers/context_dir')
+
+const metadata = require('../../../../core/metadata')
+const Buffer = require('../../../../core/local/steps/buffer')
+const incompleteFixer = require('../../../../core/local/steps/incomplete_fixer')
+
+const completedEvent = event =>
+  _.pick(event, ['_id', 'path', 'kind', 'md5sum', 'action', 'incomplete'])
```

Comment:
Shouldn't `oldPath` be included too?
E.g. what happens if we push 2 unrelated renamed events, one complete and the other incomplete?
Would the test fail if we mess up with the `oldPath` of the *complete* one?

Also what are we trying to do?
Omit `stats`? Then why not just `const omitStats = event => _.omit(event, ['stats'])`?

Reply:
I just wanted to focus on the attributes I thought were important.
Omitting `stats` would work as well.